securitycenter/notifications/receive_notifications.go

```diff
@@ -0,0 +1,45 @@
+package notifications
+
+// [START scc_receive_notifications]
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/golang/protobuf/jsonpb"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1p1beta1"
+)
+
+func parseNotificationMessage(ctx context.Context, msg *pubsub.Message) {
+	var notificationMessage = new(securitycenterpb.NotificationMessage)
+	u := &jsonpb.Unmarshaler{}
```

Comment:
Any particular reason to create an `Unmarshaler`, rather than just calling `jsonpb.Unmarshal`?

https://pkg.go.dev/github.com/golang/protobuf/jsonpb?tab=doc#Unmarshal

Reply:
No reason, switched over.